contrib/redis/go-redis.v9/option.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package redis
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+)
+
+type clientConfig struct {
+	serviceName   string
+	analyticsRate float64
+	skipRaw       bool
+}
+
+// ClientOption represents an option that can be used to create or wrap a client.
+type ClientOption func(*clientConfig)
+
+func defaults(cfg *clientConfig) {
+	cfg.serviceName = "redis.client"
+	// cfg.analyticsRate = globalconfig.AnalyticsRate()
```

Comment:
This should be removed (or implemented).

Reply:
This can be left alone to indicate that this integration does not fall under the global configuration for future developers (as opposed to other ones that have this line)